.github/actions/test-template-action/action.yml

```diff
@@ -0,0 +1,89 @@
+name: 'Test template'
+description: 'Tests MC Extension template'
+
+inputs:
+  template-name:
+    description: 'Name of the template to test'
+    required: true
+  e2e-tests-command:
+    description: 'Command to run the end-to-end tests'
+    required: true
+  typecheck-command:
+    description: 'Command to run the typecheck'
+    required: false
+  host-gcp-staging:
+    description: 'Host of the GCP staging environment'
+    required: true
+  playground-application-id:
+    description: 'Application ID of the Playground application'
+    required: true
+  mc-api-url:
+    description: 'Merchant Center API URL'
+    required: true
+  cypress-login-user:
+    description: 'Cypress login user'
+    required: true
+  cypress-login-password:
+    description: 'Cypress login password'
+    required: true
+  cypress-project-key:
+    description: 'Cypress project key'
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    # - name: Checkout
+    #   uses: actions/checkout@v4
```

Comment:
I guess this can be removed?

Reply:
Removed here: adc437d55708f9611433001be0dcff3bad79f6f8